TestGeneration.Extensions.NUnit/source.extension.vsixmanifest

```diff
@@ -6,7 +6,7 @@
     <Description xml:space="preserve">Test Generator,  NUnit extensions for Visual Studio 2015.  
 Creates Unit tests and Intellitests with both NUnit 2.6.4 and NUnit 3 frameworks.
 </Description>
-    <MoreInfo>https://github.com/nunit/nunit-vs-testgenerator/wiki</MoreInfo>
+    <MoreInfo>https://github.com/nunit/nunit-vs-testgenerator</MoreInfo>
```

Comment:
This makes sense, but should we also add the wiki link above to the Getting Started link, it is currently empty, and it is visible in VS extension lists,